src/extension_commands.ml

```diff
@@ -708,7 +706,14 @@ module MerlinJump = struct
       text_editor
       ~range:(Range.makePositions ~start:position ~end_:position)
       ~revealType:TextEditorRevealType.InCenterIfOutsideViewport
-      ()
+      ();
+    let _ =
+      Window.showTextDocument
+        ~document:(TextEditor.document text_editor)
+        ~preserveFocus:true
+        ()
+    in
+    ()
```

Comment:
👍

I think it would be cleaner to first call `showTextDocument` then wait for the resulting promise to succeed and call `set_selection` and `revealRange`.